lib/validate.ts

```diff
@@ -515,8 +515,11 @@ export async function generateExamples(
   log.filepath = options.logFilepath || log.filepath;
   for (const file of wholeInputFiles) {
     const generator = new ExampleGenerator(file, payloadDir);
-    if (operationId) {
-     await generator.generate(operationId);
+    if (operationIds) {
+     const operationIdArray = operationIds.trim().split(",");
+     for(const operationId of operationIdArray) {
```

Comment:
do you need to remove empty operationId ? for example the input is  two commas

Reply:
sounds good and ignore it.